README.md

```diff
@@ -1,32 +1,188 @@
+- Query orderbooks for all pairs within the [supported coins](https://github.com/KomodoPlatform/coins/blob/master/coins)
+- Buy/sell from the orderbook, or create maker orders
+- Configure automated ["makerbot" trading](https://developers.komodoplatform.com/basic-docs/atomicdex-api-20-dev/start_simple_market_maker_bot.html) with periodic price updates and optional [telegram](https://telegram.org/) alerts
+ 
+
+## System Requirements
 
-## How to build
+- 64-bit MacOS, Windows, or Linux operating system
+- 2GB of free RAM (or more)
+- User account with admin/root privileges
 
-1. Tools required: [Rustup](https://rustup.rs/). You will also need your OS specific build tools (e.g. build-essentials on Linux, XCode on OSX or MSVC on Win).
-1. (Optional) OSX: install openssl, e.g. `brew install openssl`.  
-1. (Optional) OSX: run `LIBRARY_PATH=/usr/local/opt/openssl/lib`
-1. Run
+## Building from source
+
+[Pre-built release binaries](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-setup/get-started-atomicdex.html) are available for OSX, Linux or Windows.
+
+If you want to build from source, the following prerequisites are required:
+- [Rustup](https://rustup.rs/)
+- [Cmake](https://cmake.org/download/) version 3.12 or higher
+- OS specific build tools (e.g. [build-essential](https://linuxhint.com/install-build-essential-ubuntu/) on Linux, [XCode](https://apps.apple.com/us/app/xcode/id497799835?mt=12) on OSX or [MSVC](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=vs-2017) on Win)
+- (Optional) OSX: install [openssl](https://www.openssl.org/), e.g. `brew install openssl`.  
+- (Optional) OSX: run `LIBRARY_PATH=/usr/local/opt/openssl/lib`
+- Additional Rust Components
     ```
     rustup install nightly-2021-05-17
     rustup default nightly-2021-05-17
     rustup component add rustfmt-preview
     ```
-1. Run `cargo build` (or `cargo build -vv` to get verbose build output).
+        
+To build, run `cargo build` (or `cargo build -vv` to get verbose build output).
+
+For more detailed instructions, please refer to the [Installation Guide](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-setup/get-started-atomicdex.html). 
+
+
+## Configuration
+
+Basic config is contained in two files, `MM2.json` and `coins`
+
+The user configuration [MM2.json file](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-setup/configure-mm2-json.html) contains rpc credentials, your mnemonic seed phrase, a `netid` (7777 is the current main network) and some extra [optional parameters](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-setup/get-started-atomicdex.html).
+
+For example:
+```json
+{
+  "gui": "core_readme",
+  "netid": 7777,
+  "rpc_password": "Ent3r_Un1Qu3_Pa$$w0rd",
+  "passphrase": "ENTER_UNIQUE_SEED_PHRASE_DONT_USE_THIS_CHANGE_IT_OR_FUNDS_NOT_SAFU",
+  "allow_weak_password": false,
```

Comment:
`allow_weak_password`, `userhome` and `dbdir` have default values. I think it's better to remove them from the example to avoid misconfiguration issues after copy-paste.